style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Times New Roman', Times, serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+  margin-right: 20px;
+}
+
+.wrap-section {
+  width: 375px;
+  height: 100%;
+  background: #fff;
+  display: flex;
+  border-radius: 0;
+  margin: auto auto;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #3c3a39;
+  height: 48px;
+  width: 375px;
+  border-radius: 0;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  padding: 0 24px;
+}
+
+header nav .my-logo {
+  color: #fff;
+  font-style: normal;
+  font-size: 18px;
+  font-family: 'Roboto', sans-serif;
+}
+
+header nav i {
+  color: #fff;
+  font-size: larger;
+  margin: px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  background: #1c1a19;
+  width: 375px;
+  height: 590px;
+  top: 48px;
+  background-image: url("assets/image/header.png");
+  background-repeat: no-repeat;
+  background-position: bottom -50px right 0;
+}
```

Comment:
[OPTIONAL] There's a strange behavior with your layout and I think it is coming from the fixed height in this class.

<img width="245" alt="Screen Shot 2022-05-27 at 10 35 16" src="https://user-images.githubusercontent.com/82426682/170731967-fdcbd9d0-a2f9-4682-9fdb-e309f27fca9e.png">

Do you see that white margin at the bottom of the page? I recommend you fix it.

Reply:
Thank you. I will fix it.